flow/project.py

```diff
@@ -1490,7 +1491,17 @@ def run(self, jobs=None, names=None, pretend=False, np=None, timeout=None, num=N
             Show a progress bar during execution.
         :type progess:
             bool
+        :param order:
+            Specify the order of operations, possible values are:
+                * None or 'none' (no reordering, do not assume any order)
+                * 'by-job'  (operations are grouped by job)
```

Comment:
Spacing of these lines is weird - it looks like they were meant to be aligned at one point. I recommend keeping it consistent at one space between the string `'by-job'` and the description `(operations are grouped by job)`, for all elements of this list.

Reply:
Done.